### PR TITLE
Add Pulsar Community Weekly Update - 2021-04-19~ 2021-04-25

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: content/2021/2021-04/2021-04-26-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+id: "2021-04-26-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2021-04-19 ~ 2021-04-25
+
+This is the Pulsar community weekly update for 2021-04-19 ~ 2021-04-25, with updates on Pulsar client, broker, transactions, and so on.
+
+### Pulsar Highlight
+
+- Add some basic supports for testing de-duplication feature.
```
Comment (Jennifer88huang) [inline content/2021/2021-04/2021-04-26-pulsar-weekly.md:15]: Do we add PR link for it?

File: content/2021/2021-04/2021-04-26-pulsar-weekly.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+- Add basic configuration for the `org.owasp:dependency-check-maven` Maven plugin and make it operational in the Apache Pulsar project.
+
+    by ([@lhotari](https://github.com/lhotari))
+
+### Development
+
+- [PIP-82] [Broker] Add support for distributed resource quota enforcement at the tenant, namespace or topic level.
+
+    https://github.com/apache/pulsar/pull/10218 ([@ravi-vaidyanathan](https://github.com/ravi-vaidyanathan))
```
Comment (Jennifer88huang) [inline content/2021/2021-04/2021-04-26-pulsar-weekly.md:35]: This PR was merged 9 days before, shall we include it in this week?

File: content/2021/2021-04/2021-04-26-pulsar-weekly.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    - 04/24: https://www.youtube.com/watch?v=JzExXJYiGvY&t=26s
+
+- Apache Pulsar Hackathon
+
+    Sign-up: https://www.eventbrite.com/e/apache-pulsar-hackathon-2021-tickets-143906003731
+
+- Pulsar Virtual Summit North America 2021
+
+    Sign-up: https://hopin.com/events/pulsar-summit-north-america-2021
+
```
Comment (Jennifer88huang) [inline content/2021/2021-04/2021-04-26-pulsar-weekly.md:85]: Reorder those events:
1. Hackathon (it's the last chance to promote this event)
2. Summit
3. Pulsar 101
4. TGIP (we don't update TGIP for a long time)

File: content/2021/2021-04/2021-04-26-pulsar-weekly.md (hunk 2 above)
Comment (timmyyuan) [reply content/2021/2021-04/2021-04-26-pulsar-weekly.md:35]: I’m not sure if we want to include it, it merged on 4.19 2:00 GMT+8

File: content/2021/2021-04/2021-04-26-pulsar-weekly.md (hunk 1 above)
Comment (Huanli-Meng) [reply content/2021/2021-04/2021-04-26-pulsar-weekly.md:15]: based on our template, Pulsar Highlight part does not cover the PR link.

**Comment by Huanli-Meng** [comment]
@Jennifer88huang please help check the comments.

File: content/2021/2021-04/2021-04-26-pulsar-weekly.md (hunk 2 above)
Comment (Jennifer88huang) [reply content/2021/2021-04/2021-04-26-pulsar-weekly.md:35]: if it's merged on 4/19, include it.

File: content/2021/2021-04/2021-04-26-pulsar-weekly.md (hunk 1 above)
Comment (Jennifer88huang) [reply content/2021/2021-04/2021-04-26-pulsar-weekly.md:15]: If we have PR or PIP to track it, it's better to add it.
You can discuss with eng and decide.

File: content/2021/2021-04/2021-04-26-pulsar-weekly.md (hunk 3 above)
Comment (Jennifer88huang) [reply content/2021/2021-04/2021-04-26-pulsar-weekly.md:85]: I suggest that we adjust the lists according to priority and timeliness. Usually, the priority of things that will happen is higher than things happened in the past. Our target is to call for users to join our upcoming events(like Hackathon / Summit, upcoming TGIP/webinar/training...) and continue to promote what happened in the past(TGIP, other recap).
If we add TGIP21 that happened last week, it belongs to video recap, the priority is lower than the upcoming events. Besides, in the TGIP link provided in the weekly, the latest one is TGIP-020(which happened on March 10th)
image

I'm open to it if you have other ideas.

Comment (Huanli-Meng) [reply content/2021/2021-04/2021-04-26-pulsar-weekly.md:85]: image

**Comment by Jennifer88huang** [comment]
@Huanli-Meng you could squash and merge the PR if you had no plan to adjust the order. 
The Hackathon signup time is past due, and it makes no sense to adjust it now.

**Comment by Huanli-Meng** [comment]
> @Huanli-Meng you could squash and merge the PR if you had no plan to adjust the order.
> The Hackathon signup time is past due, and it makes no sense to adjust it now.

wait for your approval

**Pull request merged**